package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "rethinkly",
-  "version": "1.4.2",
+  "version": "1.4.3",
   "description": "a library to make easier the operations handling in rethinkdb",
   "main": "dist/index.js",
   "scripts": {
     "example-bootstrap": "babel-node src/example/bootstrap/",
     "run-example": "babel-node src/example",
     "build": "babel src/ --out-dir dist/ --source-maps --copy-files",
     "test:ci": "ENV=mock ava",
+    "test:local": "ENV=local ava",
```

Comment:
When fixing your docker to run at 28015 you don't gonna need this script as well :D